From 4811abf2b3075aee646e8ed8b0c192ee6ff1cf50 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 08 Aug 2021 22:12:04 +0300
Subject: [PATCH] [SW-52072] gaudi: security: protect mmu_up spi msk

The mmMMU_UP_SPI_MASK register is vulnerable to random writes.
Protect it from host access.

Change-Id: Id21a34d696a5627de6b9a32dcf2f30154f7cdba1
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index c1f9333..09417e6 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -930,7 +930,6 @@
 	/* blocking regs within offsets: 0 - 0x80 */
 	prot_base = base;
 	mask = ~(PROT_BIT(mmMMU_UP_MMU_ENABLE) &
-		 PROT_BIT(mmMMU_UP_SPI_MASK) &
 		 PROT_BIT(mmMMU_UP_PAGE_ERROR_CAPTURE) &
 		 PROT_BIT(mmMMU_UP_PAGE_ERROR_CAPTURE_VA) &
 		 PROT_BIT(mmMMU_UP_ACCESS_ERROR_CAPTURE) &
